src/features/world/ui/npcs/Mayor.tsx

```diff
@@ -92,11 +117,36 @@ export const Mayor: React.FC<MayorProps> = ({ onClose }) => {
               onClose={onClose}
               message={[
                 {
-                  text: `Howdy ${username}! Seems like we've already met. In case you forgot, I'm the Mayor of this town!`, //Translate
-                },
-                {
-                  text: t("mayor.plaza.changeNamePrompt"),
+                  text: t("mayor.plaza.alreadyMet", {
```

Comment:
On an existing farm, this message popped up when I opened the plaza.

I would suggest not showing this when entering the plaza. Only when opening the modal with the Mayor - consider having separate components for the creating username (auto opens when joining plaza) and the Mayor Modal (just for changing)

<img width="558" alt="Screenshot 2025-03-26 at 11 09 10 AM" src="https://github.com/user-attachments/assets/2b28f46c-af02-441d-bb2c-9efd8b2ace40" />